rpcs3/main.cpp

```diff
@@ -187,10 +187,37 @@ struct pause_on_fatal final : logs::listener
 {
 	~pause_on_fatal() override = default;
 
-	void log(u64 /*stamp*/, const logs::message& msg, const std::string& /*prefix*/, const std::string& /*text*/) override
+	void log(u64 /*stamp*/, const logs::message& msg, const std::string& prefix, const std::string& text) override
 	{
 		if (msg.sev == logs::level::fatal)
 		{
+			std::string _msg;
```

Comment:
Better rename the struct. Maybe call it "fatal_error_handler" or something similar